### PR TITLE
Helper to decide what to keep and what to revert

### PR DESCRIPTION
**Pull request by bassosimone**
@lorenzoPrimi, as discussed, the base is the latest beta, the comparison is dev. You should basically tell me what you _don't want_ to be included. Then I'll prepare a diff and revert in dev.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,7 @@
 /ooniprobe.xcodeproj/xcuserdata
 /ooniprobe.xcworkspace/xcuserdata/
 /Pods/
+fastlane/screenshots/*/*.png
+fastlane/screenshots/screenshots.html
+fastlane/screenshots/report.xml
```
Comment (bassosimone) [inline .gitignore]: @lorenzoPrimi for example: this is from @hellais and we want to keep this bit.

File: Podfile [hunk 2]
```diff
@@ -2,7 +2,12 @@ platform :ios, '9.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 target 'ooniprobe' do
-    pod 'mkall', :git => 'https://github.com/measurement-kit/mkall-ios.git'
+  pod 'measurement_kit',
+#    :git => 'https://github.com/measurement-kit/mkall-ios.git'
+    :git => 'https://github.com/measurement-kit/ios-libs.git'
+#    pod 'measurement_kit',
+#        :git => 'https://github.com/measurement-kit/measurement-kit.git',
+#	:tag => 'v0.8.3'
```
Comment (bassosimone) [inline Podfile]: @lorenzoPrimi for example, this feels like wrong and we _don't want_ to include it.

File: Podfile [hunk 3]
```diff
@@ -2,7 +2,12 @@ platform :ios, '9.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 target 'ooniprobe' do
-    pod 'mkall', :git => 'https://github.com/measurement-kit/mkall-ios.git'
```
Comment (lorenzoPrimi) [inline Podfile]: no

File: Podfile.lock [hunk 4]
```diff
@@ -6,7 +6,7 @@ PODS:
   - Fabric (1.9.0)
   - lottie-ios (2.5.2)
   - MBProgressHUD (1.1.0)
-  - mkall (0.3.8)
```
Comment (lorenzoPrimi) [inline Podfile.lock]: no

File: ooniprobe.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -24,6 +26,8 @@
 		ED345F4F2024085E00F8DFF3 /* TestSummaryTableViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = ED345F4E2024085E00F8DFF3 /* TestSummaryTableViewController.m */; };
 		ED3582321FE948580078CCEE /* libsqlite3.tbd in Frameworks */ = {isa = PBXBuildFile; fileRef = ED3582311FE948580078CCEE /* libsqlite3.tbd */; };
 		ED3932C41E3114740064CD11 /* AboutViewController.mm in Sources */ = {isa = PBXBuildFile; fileRef = ED3932C31E3114740064CD11 /* AboutViewController.mm */; };
+		ED3AB3311D2337920061F266 /* GeoIP.dat in Resources */ = {isa = PBXBuildFile; fileRef = ED3AB32E1D2337920061F266 /* GeoIP.dat */; };
```
Comment (lorenzoPrimi) [inline ooniprobe.xcodeproj/project.pbxproj]: no

Comment (lorenzoPrimi) [reply ooniprobe.xcodeproj/project.pbxproj]: Maybe leave it, I'll modify it manually

Comment (lorenzoPrimi) [reply ooniprobe.xcodeproj/project.pbxproj]: yes definitely keep the art version

File: ooniprobe.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -1129,10 +1319,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = MADPSAYN6T;
 				ENABLE_BITCODE = NO;
-				FRAMEWORK_SEARCH_PATHS = (
```
Comment (lorenzoPrimi) [inline ooniprobe.xcodeproj/project.pbxproj]: TO SELF: change this manually

**Comment by bassosimone** [comment]
Can be closed because #213 looks good. (This PR was openned to discuss changes between different branches after a rebase brought us back of about one month; see #213)

**Pull request closed without merging**